app/uk/gov/hmrc/vatsubscription/connectors/UpdateVatSubscriptionConnector.scala

```diff
@@ -41,6 +41,7 @@ class UpdateVatSubscriptionConnector @Inject()(val http: HttpClient,
     case Api1365R5 => DESApi1365WritesR5
     case Api1365R6 => DESApi1365WritesR6
     case Api1365R7 => DESApi1365WritesR7
+
```

Comment:
Do we still need the older reads? Or do you think this could be scope creep if you remove this as well?

Reply:
Scope creep I imagine. I think we need an extra story or tech to just go through and remove them. I have no idea how much other stuff is linked to them.

Reply:
I'll remove it and see if any problems rise up. If it's not something super fast though, I'll avoid it.